### ISSUE TITLE
What files have you modified relative to the original darknet ? help

### ISSUE THREAD

**Issue by host452b** [SmitSheth/People-Counter#5]
What files have you modified relative to the original darknet ? I noticed that image.c is modified.. I want to find out  how you work this

**Comment by SmitSheth** [comment]
@clemente620 Yes, only image.c is modified, that is where the whole logic of people counting is located

**Comment by host452b** [comment]
what does image_opencv.cpp use for ?

**Comment by host452b** [comment]
there is a error that can not make

In file included from ./src/image.c:25:0:
/usr/local/include/opencv2/core/types_c.h: In function ‘cvIplImage’:
/usr/local/include/opencv2/core/types_c.h:370:12: error: incompatible types when returning type ‘int’ but ‘IplImage {aka struct _IplImage}’ was expected
     return _IplImage();
            ^
compilation terminated due to -Wfatal-errors.
Makefile:90: recipe for target 'obj/image.o' failed
make: *** [obj/image.o] Error 1

**Comment by SmitSheth** [comment]
> what does image_opencv.cpp use for ?

@clemente620 It is used for performing opencv related operations like showing, loading an image, getting frames for video,etc

**Comment by SmitSheth** [comment]
> there is a error that can not make
> 
> In file included from ./src/image.c:25:0:
> /usr/local/include/opencv2/core/types_c.h: In function ‘cvIplImage’:
> /usr/local/include/opencv2/core/types_c.h:370:12: error: incompatible types when returning type ‘int’ but ‘IplImage {aka struct _IplImage}’ was expected
> return _IplImage();
> ^
> compilation terminated due to -Wfatal-errors.
> Makefile:90: recipe for target 'obj/image.o' failed
> make: *** [obj/image.o] Error 1

@clemente620 Can you share the complete error log

**Comment by renguoyin123** [comment]
Hi,guy. There seems to be an error in " ｛｝ " of the code which you added codes in image.C. Would you please check again?@SmitSheth
